src/pages/ProfilesExplorerView/components/SceneExploreDiffFlameGraph/components/SceneComparePanel/SceneComparePanel.tsx

```diff
@@ -0,0 +1,242 @@
+    variableNames: ['profileMetricId'],
+    onReferencedVariableValueChanged: () => {
+      this.state.timeseriesPanel.updateTitle(this.buildTimeseriesTitle());
+    },
+  });
+
+  constructor({
+    target,
+    useAncestorTimeRange,
+  }: {
+    target: SceneComparePanelState['target'];
+    useAncestorTimeRange: boolean;
+  }) {
+    const filterKey = target === CompareTarget.BASELINE ? 'filtersBaseline' : 'filtersComparison';
+    const title = target === CompareTarget.BASELINE ? 'Baseline' : 'Comparison';
+    const color =
+      target === CompareTarget.BASELINE ? BASELINE_COLORS.COLOR.toString() : COMPARISON_COLORS.COLOR.toString();
+
+    super({
+      key: `${target}-panel`,
+      target,
+      filterKey,
+      title,
+      color,
+      $timeRange: new SceneTimeRange({ key: `${target}-panel-timerange` }),
+      timePicker: new SceneTimePicker({ isOnCanvas: true }),
+      refreshPicker: new SceneRefreshPicker({ isOnCanvas: true }),
+      timeseriesPanel: SceneComparePanel.buildTimeSeriesPanel({ target, filterKey, title, color }),
+    });
+
+    this.addActivationHandler(this.onActivate.bind(this, useAncestorTimeRange));
+  }
+
+  onActivate(useAncestorTimeRange: boolean) {
+    const { $timeRange, timeseriesPanel } = this.state;
+
+    if (useAncestorTimeRange) {
+      $timeRange.setState(omit(this.getAncestorTimeRange().state, 'key'));
+    }
+
+    timeseriesPanel.updateTitle(this.buildTimeseriesTitle());
+
+    const eventSub = this.subscribeToEvents();
+
+    return () => {
+      eventSub.unsubscribe();
+    };
+  }
+
+  static buildTimeSeriesPanel({ target, filterKey, title, color }: any) {
+    const timeseriesPanel = new SceneLabelValuesTimeseries({
+      item: {
+        index: 0,
+        value: target,
+        label: '',
+        queryRunnerParams: {},
+      },
+      data: new SceneDataTransformer({
+        $data: buildCompareTimeSeriesQueryRunner({ filterKey }),
+        transformations: [addRefId, addStats],
+      }),
+      overrides: (series) =>
+        series.map((s) => {
+          const metricField = s.fields[1];
+          const allValuesSum = getSeriesStatsValue(s, 'allValuesSum') || 0;
+          const formattedValue = getValueFormat(metricField.config.unit)(allValuesSum);
+          const displayName = `${title} total = ${formattedValue.text}${formattedValue.suffix}`;
+
+          return {
+            matcher: { id: FieldMatcherID.byFrameRefID, options: s.refId },
+            properties: [
+              {
+                id: 'displayName',
+                value: displayName,
+              },
+              {
+                id: 'color',
+                value: { mode: 'fixed', fixedColor: color },
+              },
+            ],
+          };
+        }),
+      headerActions: () => [new SwitchTimeRangeSelectionModeAction()],
+    });
+
+    timeseriesPanel.state.body.setState({
+      $timeRange: new SceneTimeRangeWithAnnotations({
```

Comment:
This custom time range acts as a proxy, depending on the mode it's in:
- `TimeRangeWithAnnotationsMode.DEFAULT`: forward the changes to its ancestor (the one on `SceneComparePanel `)
- `TimeRangeWithAnnotationsMode.ANNOTATIONS`: creates an annotation on the timeseries